src/Common/Core/MorganStanley.ComposeUI.Common.BasicModels/MorganStanley.ComposeUI.Common.BasicModels.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<!--
+********************************************************************************************************
+
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+
+********************************************************************************************************
+-->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\MorganStanley.ComposeUI.Common.Interfaces\MorganStanley.ComposeUI.Common.Abstractions.csproj" />
```

Comment:
In my mind, Models are associated with (ViewModels) something a bit higher then abstractions - in a sense that they have some real data and real methods changing that data.

Reply:
left it for now as we have discussed

Reply:
I'd remove the whole package/namespace. When we have actual models of something, we can decide where to put them. Since we are going to create independently usable framework packages, I'd rather put the relevant models into their respective package, eg. models of windowing/docking features will go into something like `MorganStanley.ComposeUI.Shell.Abstractions`. To me it doesn't make sense to create empty projects until we have actual code to organize.